job-dsls/jobs/FlowJob.groovy

```diff
@@ -33,7 +32,7 @@ ignore(UNSTABLE) {
     build("kieAllBuild-${kieMainBranch}", kieVersion: "$kieVersion", appformerVersion: "$appformerVersion", erraiVersionNew: "$erraiVersionNew", kieMainBranch: "$kieMainBranch")
 }
 ignore(UNSTABLE) {
-    build("prod-kieAllBuild-${kieMainBranch}", kieProdVersion: "$kieProdVersion", appformerProdVersion: "$appformerProdVersion", erraiProdVersion: "$erraiProdVersion", kieMainBranch: "$kieMainBranch", \
+    build("prod-kieAllBuild-${kieMainBranch}", kieProdVersion: "$kieProdVersion", appformerProdVersion: "$appformerProdVersion", erraiVersion: "$erraiVersionNew", kieMainBranch: "$kieMainBranch", \
```

Comment:
where is defined `$erraiVersionNew` ?

Reply:
it is defined here
https://github.com/mbiarnes/kie-jenkins-scripts/blob/9e38b3904f13064d8b26ad17cf9cc42e5fc025d5/job-dsls/jobs/FlowJob.groovy#L21

Reply:
coming from here:
https://github.com/mbiarnes/kie-jenkins-scripts/blob/9e38b3904f13064d8b26ad17cf9cc42e5fc025d5/job-dsls/jobs/FlowJob.groovy#L21